staging/src/k8s.io/client-go/features/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - feature-approvers
```

Comment:
that's a large number of ppl. Would not expect many of them owning features here.

Reply:
would you prefer to remove this file and rely on the top-level one ?

Reply:
yes, between the two options the client-go approvers is the better set of ppl.

Reply:
done.